### ISSUE TITLE
Plugin doesn't seem to honor character limit and there seems to be a bug with the character limit when deleting characters

### ISSUE THREAD

**Issue by carlofabyss** [msaari/relevanssi-live-ajax-search#14]
Hello, thank you for creating this plugin.  There have been very few issues so far.  But as the title states, it seems that the plugin doesn't seem to honor the character limit set in relevanssi and seems to work independepently with a hard limit of 3 characters.  This on its own is not that big of a deal.  I did try to set the filter as follows per relevanssi's documentation:

add_filter( 'relevanssi_block_one_letter_searches', '__return_false' );

And I also tried to set the character limit within relevanssi's settings but that seemed to not have any effect on this plugin's limit.  I can also see that you addressed this part of it in another issue that was closed so no need to address this part.

There is however something that seems to be a bigger deal.  When I go to backspace in my search bar and the letters go under 3, the spinner just seems to stay there forever and never goes away.  Indicating that there is something wrong with how it handles the character limit when the string is changed.  However, the limit seems fine when initially typing in the box.  I did try to remove my custom js just to rule it out and there is no change in how it behaves.

It would be good to know if you are able to duplicate this or not, just so I can rule out whether its caused by something third party or if its a bug with the plugin itself.  Thanks for you help and guidance.  I am on the latest version 2.3.1.

**Comment by msaari** [comment]
Live Ajax Search has a separate character limit (because it can be used independently without Relevanssi). Ideally, that would be the same as in Relevanssi,  but it's possible to have different character limits in the plugins. In both plugins, the default value is 3. Would it make sense to sync these values? I guess so, but I can also see cases where the user might want to have a different minimum character count in both searches. But it never makes sense to have a lower character limit in the Live Ajax Search, so blocking that would probably make sense.

I can confirm this: if I start typing, nothing happens until I type the third letter (as the minimum character count is the default three on my site), and then, when I backspace, the spinner remains spinning as the search is waiting for the third character to search again. Once I do that, the spinner disappears, and the search results appear.

It's not a huge problem, but also easy to fix.

**Comment by carlofabyss** [comment]
Thank you, the snippet in the other closed post resolved my issue with the spinner continuing to show when backspacing since it is no longer waiting on the third character.
